### PR TITLE
Add iot-tenant-cleaner image

### PR DESCRIPTION
**Pull request by ctron**

### CODE REVIEW

File: iot-tenant-cleaner/image.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+schema_version: 1
+
+version: 1.4
+name: amq7/amq-online-1-iot-tenant-cleaner
```
Comment (lulf) [inline iot-tenant-cleaner/image.yaml]: Needs the -rhel7 suffix (this is needed for all new images requested in comet)

File: iot-tenant-cleaner/override-dev.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+osbs:
+      repository:
+            name: containers/amq-online-1
+            branch: amq7-amq-online-1-iot-tenant-cleaner-dev-rhel-7
```
Comment (lulf) [inline iot-tenant-cleaner/override-dev.yaml:4]: Remove this file, it is no longer used.

**Review by lulf — CHANGES_REQUESTED** [review]
Two minor nits, otherwise good!

File: iot-tenant-cleaner/override-dev.yaml (hunk 2 above)
Comment (ctron) [reply iot-tenant-cleaner/override-dev.yaml:4]: I saw it in the other directories as well

File: iot-tenant-cleaner/image.yaml (hunk 1 above)
Comment (ctron) [reply iot-tenant-cleaner/image.yaml]: I copied that from the tenant service, and it is missing there as well … I will still add it here.

File: iot-tenant-cleaner/override-dev.yaml (hunk 2 above)
Comment (lulf) [reply iot-tenant-cleaner/override-dev.yaml:4]: Hmm, yeah. We can either remove those, or modify this one to contain the correct information. The name and branch should be set to the name and branch of the comet build repo, which I don't think you've requested yet.

File: iot-tenant-cleaner/image.yaml (hunk 1 above)
Comment (lulf) [reply iot-tenant-cleaner/image.yaml]: Existing images does not need to have this suffix, its only required for new images.

**Pull request merged**